### ISSUE TITLE
Pulumi.Aws.CloudWatch.EventRule escape > with \u003e

### ISSUE THREAD

**Issue by guitarrapc** [pulumi/pulumi-aws#1009]
## TL;DR

Pulumi.Aws.CloudWatch.EventRule always escape `>` as unicode `\u003e` on serialization.
This prevent user to put EventPattern when they want filter event with.

## Runtime

.NET Core / C#

## Reproduce

Event Pattern can pass `detail-type` to filter event, like follows.
I expected to put this JSON as is, however Serialization seems espcaping `>` and similar operator char.

```csharp
var eventRule = new EventRule($"guardduty-eventrule", new EventRuleArgs
{
    Name = $"guardduty-eventrule",
    EventPattern = @"{""detail"":{ ""severity"":[{""numeric"":["">="", 4]}]},""detail-type"":[""GuardDuty Finding""],""source"":[""aws.guardduty""] }",
});
```

## Expected result

EventRule EventPattern should contains JSON as is.

```json
{"detail":{"severity":[{"numeric":[">=",4]}]},"detail-type":["GuardDuty Finding"],"source":["aws.guardduty"]}
```

## Actual result

`>` escape to unicode `\u003e` and break event pattern.

```json
{"detail":{"severity":[{"numeric":["\u003e=",4]}]},"detail-type":["GuardDuty Finding"],"source":["aws.guardduty"]}
```

![image](https://user-images.githubusercontent.com/3856350/84890058-d74e2700-b0d4-11ea-8c3e-2aaef3fac239.png)

## Version

```csproj
    <PackageReference Update="Pulumi.Aws" Version="2.9.1" />
    <PackageReference Update="Pulumi" Version="2.4.0" />
```

## NOTE

It seems like [current EventRule usecase](https://github.com/pulumi/pulumi-aws/blob/11b3ffed199a9d438cff893f81c61f6321928cc9/sdk/dotnet/CloudWatch/EventTarget.cs#L19-L74) not consider `detail-type` include `>` or `<` char. Serialization on Pulumi repo may not specifying encoder `JavaScriptEncoder.UnsafeRelaxedJsonEscaping`?

**Comment by guitarrapc** [comment]
I've check with terraform resource and it reproduce with 2.67.0. It's not Pulumi causing.

```terraform
resource "aws_cloudwatch_event_rule" "main" {
  name        = "guardduty-eventrule"
  event_pattern = <<PATTERN
{
  "source": ["aws.guardduty"],
  "detail-type": [
    "GuardDuty Finding"
  ],
  "detail" : {
    "severity":[{"numeric":[">=",4]}]
  }
}
PATTERN
}
```

```json
{"detail":{"severity":[{"numeric":["\u003e=",4]}]},"detail-type":["GuardDuty Finding"],"source":["aws.guardduty"]}
```

AWS docs describe to specify each severity.

> https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings_cloudwatch.html

Let's close issue.
